Nodejs/Product/TestAdapter/TestFrameworks/find_tests.js

```diff
@@ -23,5 +26,4 @@ try {
 } catch (exception) {
     console.log("NTVS_ERROR:TestFramework (" + process.argv[2] + ") threw an exception processing (" + process.argv[3] + "), " + exception);
     process.exit(1);
-}
-process.exit(0);
```

Comment:
Rather than deleting this, would it make sense for each framework to specify whether they're sync or async and, in the former case, do this, but in the latter case, explicitly exit when the promise resolves? This might be fine, I was just a little surprised that find_tests was suddenly returning a promise, since the calling pattern doesn't suggest that as the case at all. I also think without handling that, you might not be catching exceptions it's throwing in the same way that you are for the synchronous frameworks.